### PR TITLE
Feature/getting started examples

### PR DESCRIPTION
**Pull request by SBroecker**
## Description

Formatting and wording changes to the three getting started examples

### CODE REVIEW

File: examples/classification_eval.ipynb
Comment (mdelrosa) [inline examples/classification_eval.ipynb:1]: These updates are great -- the new markdown cells/comments add important context for first-time users. 

One Nit: Change "# Generate a scenario set" to "# Define a scenario set" -- "generate" has a specific meaning in the context of Okareo that doesn't really apply here.

File: examples/retrieval_eval.ipynb
Comment (mdelrosa) [inline examples/retrieval_eval.ipynb:1]: Nit: delete the last cell of the notebook since it contains no code/markdown

**Review by mdelrosa — APPROVED** [review]
Looks great! Thanks for doing this. Just two small nits -- otherwise looks good to merge

**Comment by SBroecker** [comment]
Good catch on those! Fixed

**Pull request merged**